generic-templates/newsletter-signup-snippet/snippet.html

```diff
@@ -1,1491 +1,1516 @@
 
-          <option value="VC">
-            Saint Vincent and the Grenadines
-          </option>
+              <option value="WS">
+                Samoa
+              </option>
 
-          <option value="WS">
-            Samoa
-          </option>
+              <option value="SM">
+                San Marino
+              </option>
 
-          <option value="SM">
-            San Marino
-          </option>
+              <option value="ST">
+                Sao Tome and Principe
+              </option>
 
-          <option value="ST">
-            Sao Tome and Principe
-          </option>
+              <option value="SA">
+                Saudi Arabia
+              </option>
 
-          <option value="SA">
-            Saudi Arabia
-          </option>
+              <option value="SN">
+                Senegal
+              </option>
 
-          <option value="SN">
-            Senegal
-          </option>
+              <option value="RS">
+                Serbia
+              </option>
 
-          <option value="RS">
-            Serbia
-          </option>
+              <option value="SC">
+                Seychelles
+              </option>
 
-          <option value="SC">
-            Seychelles
-          </option>
+              <option value="SL">
+                Sierra Leone
+              </option>
 
-          <option value="SL">
-            Sierra Leone
-          </option>
+              <option value="SG">
+                Singapore
+              </option>
 
-          <option value="SG">
-            Singapore
-          </option>
+              <option value="SK">
+                Slovakia
+              </option>
 
-          <option value="SK">
-            Slovakia
-          </option>
+              <option value="SI">
+                Slovenia
+              </option>
 
-          <option value="SI">
-            Slovenia
-          </option>
+              <option value="SB">
+                Solomon Islands
+              </option>
 
-          <option value="SB">
-            Solomon Islands
-          </option>
+              <option value="SO">
+                Somalia
+              </option>
 
-          <option value="SO">
-            Somalia
-          </option>
+              <option value="ZA">
+                South Africa
+              </option>
 
-          <option value="ZA">
-            South Africa
-          </option>
+              <option value="SS">
+                South Sudan
+              </option>
 
-          <option value="SS">
-            South Sudan
-          </option>
+              <option value="ES">
+                Spain
+              </option>
 
-          <option value="ES">
-            Spain
-          </option>
+              <option value="LK">
+                Sri Lanka
+              </option>
 
-          <option value="LK">
-            Sri Lanka
-          </option>
+              <option value="SH">
+                St. Helena
+           
```

Comment:
Yeah, been in since FF10. http://caniuse.com/#search=3d%20transforms

However, I didn't actually need the 3d version for what I was doing, so moved to 2d and now it's all the way back to version 3.5 it seems: http://caniuse.com/#feat=transforms2d